scripts/runner-update-env.js

```diff
@@ -73,7 +73,10 @@ async function fetchRunners() {
 
         services = services.concat(filteredServices);
 
-        await sleep(1000);
+        const remaining = parseInt(response.headers.get('ratelimit-remaining'));
+        const resetMs = (parseInt(response.headers.get('ratelimit-reset')) + 1) * 1000;
+
+        await sleep(Math.ceil((resetMs + 1) / remaining));
```

Comment:
why this `+ 1`?

Reply:
Rate limit remaining time usually doesn't take the time it takes for the request to arrive on the other side into account, so it's usually a good idea to give it some head room. This will divide across their rate limit utilization so it should be something like an additional 10ms per request on average.

Reply:
Oh, I see now. I had a redundant one in there. Removed in 1da6c41b